grpc-interface-support/src/conan_helper.py

```diff
@@ -0,0 +1,185 @@
+
+    headers_relative, sources_relative = move_generated_sources(
+        project_dir, project_dir, include_dir, src_dir
+    )
+
+    files_to_copy = [
+        CopySpec(source_path="CMakeLists.txt"),
+        CopySpec(source_path="conanfile.py"),
+        CopySpec(
+            "ServiceNameServiceClientFactory.h",
+            f"{include_dir}/{to_camel_case(service_name)}ServiceClientFactory.h",
+        ),
+        CopySpec(
+            "ServiceNameServiceClientFactory.cc",
+            f"{src_dir}/{to_camel_case(service_name)}ServiceClientFactory.cc",
+        ),
+    ]
+
+    headers_relative.append(
+        f"{include_dir}/{to_camel_case(service_name)}ServiceClientFactory.h"
+    )
+    sources_relative.append(
+        f"{src_dir}/{to_camel_case(service_name)}ServiceClientFactory.cc"
+    )
+
+    variables = {
+        "service_name": service_name,
+        "service_name_lower": service_name.lower(),
+        "service_name_camel_case": to_camel_case(service_name),
+        "headers": "\n\t".join(headers_relative),
+        "sources": "\n\t".join(sources_relative),
+        "package_id": interface_namespace.replace(".", "::"),
+        "core_sdk_version": str(get_required_sdk_version()),
+    }
+
+    template_dir = os.path.join(
+        get_package_path(), "grpc-interface-support", "templates", "cpp"
+    )
+
+    copy_templates(template_dir, project_dir, files_to_copy, variables)
+
+
+def export_conan_project(conan_project_path: str) -> None:
+    """Export a conan project to the local conan cache.
+
+    Args:
+        conan_project_path (str): The path to directory containing the project.
+    """
+    print("Exporting Conan project")
+    subprocess.check_call(
+        ["conan", "export", "."],
+        cwd=conan_project_path,
+        stdout=subprocess.DEVNULL,
+        stderr=subprocess.DEVNULL,
+    )
+
+
+def add_dependency_to_conanfile(dependency_name: str) -> None:
+    """Add the dependency name to the project's list of dependencies.
+
+    Args:
+        dependency_name (str): The dependency to add e.g. grpc@1.50.1
+    """
+    conanfile_path = os.path.join(get_workspace_dir(), "conanfile.txt")
+
+    in_requires_section = False
+    lines = []
+    requirements: List[str] = []
+    with open(conanfile_path, encoding="utf-8", mode="r") as conanfile:
+        for line in conanfile:
+            if line.strip() == "[requires]":
+                in_requires_section = True
+            elif in_requires_section and line.strip().startswith("["):
+                in_requires_section = False
+
+                if dependency_name not in requirements:
+                    lines.append(dependency_name)
+                    lines.append("\n")
+
+            if in_requires_section:
+                if len(line.strip()) > 0:
+                    requirements.append(line.strip())
+
+            lines.append(line)
+
+    if in_requires_section and dependency_name not in requirements:
```

Comment:
```suggestion
    # Check adding dependency in case [requires] section is last section of file
    if in_requires_section and dependency_name not in requirements:
```